lib/src/package_analysis/shapes_ext.dart

```diff
@@ -60,3 +60,19 @@ extension LibraryShapeExt on LibraryShape {
         exportedClasses: exportedClasses ?? this.exportedClasses,
       );
 }
+
+extension PackageShapeExt on PackageShape {
+  /// Returns a list of the names of all the top-level functions which are defined in this package.
+  List<String> get getFunctions {
+    return functions.map((function) => function.name).toList();
+  }
```

Comment:
Don't use `get...` names..
```suggestion
  List<String> get functionNames {
    return functions.map((function) => function.name).toList();
  }
```